### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes https://github.com/jsoref/flank/actions/runs/11876519325#summary-33094938650

Note that in addition to spelling fixes, I've included a set of accessibility fixes ...

For more information, see:
* https://www.w3.org/QA/Tips/noClickHere
* https://webaim.org/techniques/hypertext/link_text
* https://granicus.com/blog/why-click-here-links-are-bad/
* https://heyoka.medium.com/dont-use-click-here-f32f445d1021

I'm happy to split them to a distinct PR (or drop them or anything else).

There's one item which is almost certainly an API break. As with everything else, it can be split out or dropped.

https://github.com/jsoref/flank/actions/runs/11876519448#summary-33094938868

## Checklist

- [ ] Documented
- [x] Unit tested
- [ ] Integration tests updated

### CODE REVIEW

File: .github/workflows/macos_workflow.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ on:
     branches:
       - '*'
 
-permissions:  # limit the permissions of the GITHIUB_TOKEN to reading repository contents
+permissions:  # limit the permissions of the GITHUB_TOKEN to reading repository contents
```
Comment (jsoref) [inline .github/workflows/macos_workflow.yml:11]: I'm deleting this file in another PR...

File: .github/workflows/update_dependencies.yml [hunk 2]
```diff
@@ -101,7 +101,7 @@ jobs:
           body: "Firebase Api update"
           labels: |
             automated pr
-            firease_api
+            firebase_api
```
Comment (jsoref) [inline .github/workflows/update_dependencies.yml:104]: this is a change that something might notice

File: flank-scripts/src/test/kotlin/flank/scripts/ops/common/ConventionalCommitFormatterTest.kt [hunk 3]
```diff
@@ -19,7 +19,7 @@ class ConventionalCommitFormatterTest(
             arrayOf("feat: Sample", "Features", "Sample"),
             arrayOf("fix: Bug", "Bug Fixes", "Bug"),
             arrayOf("docs: New", "Documentation", "New"),
-            arrayOf("refactor: Eveerything", "Refactor", "Eveerything"),
+            arrayOf("refactor: Everything", "Refactor", "Everything"),
```
Comment (jsoref) [inline flank-scripts/src/test/kotlin/flank/scripts/ops/common/ConventionalCommitFormatterTest.kt:22]: This repository didn't seem particularly playful...

File: test_runner/build.gradle.kts [hunk 4]
```diff
@@ -347,7 +347,7 @@ tasks.test {
     minHeapSize = "512m"
     dependsOn(":resolveArtifacts")
 
-    // deletes 'results' directory which contains remainings from unit tests
+    // deletes 'results' directory which contains remains from unit tests
```
Comment (jsoref) [inline test_runner/build.gradle.kts:350]: alternatively `remnants`

File: test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt [hunk 5]
```diff
@@ -2193,7 +2193,7 @@ AndroidArgs
             - model: blueline
               version: 28
               locale: en
-              orientation: poortrait
+              orientation: portrait_
```
Comment (jsoref) [inline test_runner/src/test/kotlin/ftl/args/AndroidArgsTest.kt:2196]: I prefer to use `_` to indicate "this isn't the real word" while still making it easy for spell checking tools to find typos. But, I'm happy to drop these changes...

File: test_runner/src/test/kotlin/ftl/args/yml/test_error_yaml_cases/flank-no-device-version.yml [hunk 6]
```diff
@@ -18,7 +18,7 @@ gcloud:
       orientation: portrait
       # Samsung Galaxy S9 SM-G9600
     - model: starqltechn
-      versxion: 28
+      version_: 28
```
Comment (jsoref) [inline test_runner/src/test/kotlin/ftl/args/yml/test_error_yaml_cases/flank-no-device-version.yml:21]: again, it's helpful to show "yes, we're trying not to use the real word" vs "oops, we didn't mean to make a typo, but here it is".

File: tool/execution/linear/src/main/kotlin/flank/execution/linear/Transform.kt [hunk 7]
```diff
@@ -1,4 +1,4 @@
-package flank.exection.linear
+package flank.execution.linear
```
Comment (jsoref) [inline tool/execution/linear/src/main/kotlin/flank/execution/linear/Transform.kt:1]: I'm renaming this entire package / corresponding directories. I expect this is a major api break. It seems like an odd spelling. Happy to drop...

File: .github/workflows/update_dependencies.yml [hunk 8]
```diff
@@ -101,7 +101,7 @@ jobs:
...
           reviewers: adamvduke,jakeouellette,tonybaroneee
```
Comment (kaibolay) [inline .github/workflows/update_dependencies.yml:105]: please replace `adamvduke` with `kaibolay`

File: tool/execution/linear/src/main/kotlin/flank/execution/linear/Transform.kt (hunk 7 above)
Comment (kaibolay) [reply tool/execution/linear/src/main/kotlin/flank/execution/linear/Transform.kt:1]: Let's just fix it while we're at it.

**Review by kaibolay — APPROVED** [review]

File: .github/workflows/update_dependencies.yml (hunk 8 above)
Comment (jsoref) [reply .github/workflows/update_dependencies.yml:105]: There are 3 instances, do you want all 3 changed, or just this one?

https://github.com/Flank/flank/blob/7a9163b2aba43102da960ffbedf612dc114a93af/.github/workflows/generate_release_notes.yml#L58
https://github.com/Flank/flank/blob/7a9163b2aba43102da960ffbedf612dc114a93af/.github/workflows/update_dependencies.yml#L60
https://github.com/Flank/flank/blob/7a9163b2aba43102da960ffbedf612dc114a93af/.github/workflows/update_dependencies.yml#L105

Comment (kaibolay) [reply .github/workflows/update_dependencies.yml:105]: All of them, please.

**Review by kaibolay — DISMISSED** [review]

**Review by kaibolay — APPROVED** [review]

**Pull request merged**